gui/l10n/AbracaDABra_de.ts

```diff
@@ -3583,22 +3583,22 @@ Datei: %1</translation>
     <message>
         <location filename="../setupdialog.ui" line="1363"/>
         <source>Dynamic label (DL) messages are recorded in TXT file during audio recording when this option is enabled.</source>
-        <translation type="unfinished"></translation>
+        <translation>Dynamic-label-(DL)-Meldungen werden während der Audioaufnahme in der TXT Datei gespeichert wenn diese Option aktiviert ist.</translation>
```

Comment:
oops, I forgot a comma, should be "gespeichert, wenn"

Reply:
No problem, I will fix comma after merging. Thanks for quick update!